jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.vue

```diff
@@ -35,37 +35,36 @@
       ></v-text-field>
     </v-row>
 
-    <v-div v-if="n_moment > 0 && dataset_spectral_unit !== ''">
+    <div v-if="n_moment > 0 && dataset_spectral_unit !== ''">
       <v-row>
         <v-radio-group
           label="Output Units"
           hint="Choose whether calculated moment is in units of wavelength or velocity."
           v-model="output_unit_selected"
-          column>
+          row
```

Comment:
I did want this as column so that the options stay stacked under the heading no matter what width the tray is. With this the heading+choices wrap, which I don't like:

https://github.com/rosteen/jdaviz/assets/39831871/29208e12-1719-491e-91d3-7ace5e296874

Reply:
ok, I guess that just doesn't happen for links control with shorter strings.  Someday we can force the heading on its own line and stack them, but I don't think its necessary for now.